src/validators/Validator.js

```diff
@@ -0,0 +1,28 @@
+export default class Validator {
+  #synonyms
+  #re
```

Comment:
I have to read more code to understand this.
You can solve this by a docblock, or write it full like: `#regularExpression`

Reply:
docblocks are on the wishlist